org.opennumismat.OpenNumismat.yml

```
@@ -0,0 +1,47 @@
+app-id: org.opennumismat.OpenNumismat
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: open-numismat
+rename-icon: open-numismat
+rename-desktop-file: open-numismat.desktop
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '5.15-21.08'
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=host
+  - --device=dri
+  - --env=QT_QPA_PLATFORM=xcb
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - python3-requirements.json
+  - name: opennumismat
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --no-deps .
+      - install -Dm644 open-numismat.desktop /app/share/applications/open-numismat.desktop
+      - install -Dm644 open-numismat.png /app/share/icons/hicolor/48x48/apps/open-numismat.png
+    sources:
+      - type: git
+        url: https://github.com/OpenNumismat/open-numismat.git
+        tag: 1.8.15.1
+        commit: 6e15d747b1a6b9c473077c4d5a3b34219282ff31
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/OpenNumismat/open-numismat/releases/latest
+          tag-query: .tag_name
+          version-query: $tag
+          timestamp-query: .published_at
+  - name: metadata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 org.opennumismat.OpenNumismat.appdata.xml /app/share/appdata/org.opennumismat.OpenNumismat.appdata.xml
+    sources:
+      - type: file
+        path: org.opennumismat.OpenNumismat.appdata.xml
```

```suggestion
      - install -Dm644 open-numismat.png /app/share/icons/hicolor/48x48/apps/open-numismat.png
      - install -Dm644 org.opennumismat.OpenNumismat.appdata.xml /app/share/appdata/org.opennumismat.OpenNumismat.appdata.xml
    sources:
      - type: git
        url: https://github.com/OpenNumismat/open-numismat.git
        tag: 1.8.15.1
        commit: 6e15d747b1a6b9c473077c4d5a3b34219282ff31
        x-checker-data:
          type: json
          url: https://api.github.com/repos/OpenNumismat/open-numismat/releases/latest
          tag-query: .tag_name
          version-query: $tag
          timestamp-query: .published_at
      - type: file
        path: org.opennumismat.OpenNumismat.appdata.xml
```


Also the appdata should be submitted upstream so it can be managed there. (it's not flatpak specific)